kafka-rest/src/test/java/io/confluent/kafkarest/integration/v3/ClusterConfigsResourceIntegrationTest.java

```diff
@@ -0,0 +1,250 @@
+package io.confluent.kafkarest.integration.v3;
+
+import static java.util.Collections.singletonList;
+import static junit.framework.TestCase.assertEquals;
+
+import io.confluent.kafkarest.Versions;
+import io.confluent.kafkarest.entities.ClusterConfig;
+import io.confluent.kafkarest.entities.ConfigSource;
+import io.confluent.kafkarest.entities.v3.BrokerConfigData;
+import io.confluent.kafkarest.entities.v3.ClusterConfigData;
+import io.confluent.kafkarest.entities.v3.ConfigSynonymData;
+import io.confluent.kafkarest.entities.v3.GetBrokerConfigResponse;
+import io.confluent.kafkarest.entities.v3.GetClusterConfigResponse;
+import io.confluent.kafkarest.entities.v3.ResourceLink;
+import io.confluent.kafkarest.integration.ClusterTestHarness;
+import java.util.Arrays;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ClusterConfigsResourceIntegrationTest extends ClusterTestHarness {
+
+  public ClusterConfigsResourceIntegrationTest() {
+    super(/* numClusters= */ 3, /* withSchemaRegistry= */ false);
+  }
+
+  @Test
+  public void listClusterConfigs_nonExistingCluster_throwsNotFound() {
```

Comment:
Shall we add test for listing cluster configs for an existing cluster ?

Reply:
Acknowledged.

Solved on #686 (want to avoid the merge conflict).